include/boost/numeric/odeint/stepper/adaptive_adams_bashforth.hpp

```diff
@@ -89,6 +89,18 @@ class adaptive_adams_bashforth: public algebra_stepper_base< Algebra , Operation
         do_step_impl(m_coeff, in, t, out, dt);
     };
 
+    template<class System>
+    void initialize(System system, state_type &inOut, time_type &t, time_type dt)
+    {
+        m_coeff.reset();
+
+        for(size_t i=0; i<steps; ++i)
+        {
+            do_step(system, inOut, t, dt);
```

Comment:
Would be possible, yeah. Another possibility would be to make use of an initializing stepper (same as done in adams_bashforth)